code/game/objects/structures/crates_lockers/closets/secure/security.dm

```diff
@@ -80,6 +80,7 @@
 	new	/obj/item/storage/firstaid/o2(src)
 	new	/obj/item/storage/firstaid/brute(src)
 	new /obj/item/storage/belt/medical(src)
+	new /obj/item/storage/belt/security/full(src)
```

Comment:
please dont give them validhunting gear. actually, it'd be a better idea to remove that fucking locker room access entirely

Reply:
Imagine trying to validhunt with only a baton. They have no gun, have everything else anyway, this change just means there's always enough for when all job slots are full.

Reply:
a secbelt is full of gear designed for validhunting. give them deputy tools, maybe

Reply:
I agree with zesko here

Reply:
Alright swapped for deputy belt